notebooks/checks/integrity/dominant_frequency_change.ipynb

```diff
@@ -158,7 +158,7 @@
     }
    ],
    "source": [
-    "check.run(dataset=validation_dataset, baseline_dataset=train_dataset)"
+    "check.run(validation_dataset, train_dataset)"
```

Comment:
switch here as well, to match roles to name

Also (again not your code but it would help) - rename validation to test